.github/workflows/docs.yml

```diff
@@ -0,0 +1,38 @@
+name: Generating API documentation
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'api-reference'
```

Comment:
@Sushisource should we rebuild the documentation on every push to `master`, or is it better to configure it for stable tags?